### PR TITLE
One logger per module, and some associated changes:

### LINKED ISSUE #62: One logger per module

### PR DESCRIPTION
**Pull request by julia326**
- all print statements are now logs
- some infos are now debugs, was too verbose before
- not adding a logging.config yet, may add later if mhctools often used as binary

Fixes #62, #68 

---

This change is [Reviewable](https://reviewable.io/reviews/hammerlab/mhctools/63)

### CODE REVIEW

**Comment by ihodes** [comment]
LGTM--I'd try rerunning Travis tests tomorrow, looks like IEDB was down?

File: mhctools/cli/script.py [hunk 1]
```diff
@@ -57,6 +62,6 @@ def main(args_list=None):
     }
     epitope_collection = predictor.predict(input_dictionary)
     df = epitope_collection.to_dataframe()
-    print(df)
```
Comment (iskandr) [inline mhctools/cli/script.py:60]: Printing this DataFrame is sometimes the only output I care about (e.g. when checking if there are any strong MHC binding subsequences within one particular sequence), so I'd rather leave this as a print (or at least promote the log statement to `info`).

File: mhctools/process_helpers.py [hunk 2]
```diff
@@ -108,8 +112,10 @@ def run_multiple_commands_redirect_stdout(
 
     def add_to_queue(process):
         if print_commands:
-            print(" ".join(process.args), ">",
-                  process.redirect_stdout_file.name)
+            handler = logging.FileHandler(process.redirect_stdout_file.name)
+            handler.setLevel(logging.DEBUG)
+            logger.addHandler(handler)
```
Comment (iskandr) [inline mhctools/process_helpers.py:117]: It seems like the logger could accumulate many handlers from dead processes (sometimes this code runs tens of thousands of times), do you think that will be a problem?

Comment (julia326) [reply mhctools/process_helpers.py:117]: so in that situation, does it currently write to tens of thousands of files? or it runs 20k times but only writes to a handful of files?

Comment (julia326) [reply mhctools/process_helpers.py:117]: adding a line that removes the handler after the debug statements get written

**Pull request merged**